code/modules/reagents/chemistry/reagents/drinks.dm

```diff
@@ -146,6 +158,7 @@
 	drink_icon = "banana"
 	drink_name = "Glass of banana juice"
 	drink_desc = "The raw essence of a banana. HONK"
+	taste_message = "banana juice"
```

Comment:
I'd say "honk" but you're already using it somewhere. Doesn't mean you can again though